Automattic/jetpack
# Make sidebar jitms dismissable

Make just-in-time-messages displayed in the sidebar for wpcom simple and atomic sites be able to be dismissed.



Part of [wp-calypso#52368](https://github.com/Automattic/wp-calypso/issues/52368)

#### Changes proposed in this Pull Request:

*

#### Jetpack product discussion



#### Does this pull request change what data or activity we track or use?



Dismissals from calypso are already recorded, dismissals from jetpack sites are already recorded, this is doing the same thing for simple and atomic sites.

#### Testing instructions:






1. Load a wp-admin page, you should see the JITM displayed in the sidebar, if not then:
   * set-up your wpcom sandbox to test jitms following these instructions: PCYsg-cLC-p2
   * Modify some of the wpcom JITM messages so that they're eligible for display on your site
2. Apply this PR to your test environment 
3. Mark the JITM as dismissible by changing it in `jitm-engines.php` on your sandbox to include a call to `is_dismissible( true )`
4. Reload the page
5. The JITM should now have an X to dismiss it like this: 
![image](https://user-images.githubusercontent.com/93301/143612026-a3bd722f-1a0b-46c4-b67a-3459e718fb96.png)
6. Click the X
7. The JITM should disappear
8. Reload the page, the JITM should still not be displayed (though a different one may be)

Repeat it again on an atomic site (extra hoops to jump, see PCYsg-cLC-p2)
